### PR TITLE
Switch to using static libs on Windows

### PR DESCRIPTION
**Pull request by tustanivsky**
Switching to the static version of `sentry-native` libs on Windows should allow bypassing issues with publishing to UE Marketplace (obfuscation of certain binaries required by Sentry plugin).

Related to #424

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -33,13 +33,13 @@ jobs:
     uses: ./.github/workflows/sdk.yml
     with:
       target: Win64-Crashpad
-      runsOn: windows-latest
+      runsOn: windows-2019
```
Comment (bitsandfoxes) [inline .github/workflows/ci.yml:36]: Why is that necessary?

**Review by bitsandfoxes — COMMENTED** [review]
Why do we need specifically `VS 2019` to build sentry-native?

**Comment by tustanivsky** [comment]
> Why do we need specifically `VS 2019` to build sentry-native?

In order to preserve the [compatibility](https://docs.unrealengine.com/5.3/en-US/setting-up-visual-studio-development-environment-for-cplusplus-projects-in-unreal-engine/) of `sentry-native` static libs with Unreal we have to stick to VS 2019. Afaik VS 2019 version remains available only at `windows-2019` VMs so that's why we have to hold up with using `windows-latest` at this point.

**Review by bitsandfoxes — APPROVED** [review]
